python/solutions/part1/longest_word_finder.py

```diff
@@ -0,0 +1,9 @@
+sentence_string = "This is a long sentence with words of multiple lengths for inputting into your function"
+longest_word = ''
+
+words = sentence_string.split()
```

Comment:
I see no difference in clarity, just a stylistic preference. Mine is to declare variables first before doing any work, so this is how I would write it. You may change it if you want.